app/src/main/java/eu/theinvaded/mastondroid/viewmodel/LoginViewModelContract.java

```diff
@@ -20,7 +20,7 @@
         String getClientSecret();
         String getClientId();
         void setUser(MastodonAccount account);
-        void showLoginError();
+        void showLoginError(Throwable loginError);
```

Comment:
I have no particular opinion about that. Care to refactor that and link to your idea of how it should look?